docs/configuration/index.md

```diff
@@ -893,8 +897,15 @@ The `epel-all` alias includes all active minor versions of all active major EPEL
 
 :::info
 
-For Copr builds all EPEL 10 minor version aliases are currently mapped to the `epel-10` target.
-This is subject to change.
+Unlike `epel-10`, which is mapped to `CentOS-Stream-10` Testing Farm compose by default,
+there is no default mapping for the `rhel+epel-10` Copr chroot. You have to specify
+a compose explicitly, e.g.:
+
+```
+  targets:
+    rhel+epel-10-x86_64:
+      distros: [CentOS-Stream-10]
```

Comment:
Does it make sense? Should I perhaps add a different (and more sensible) example with `use_internal_tf: true` and some `RHEL-10.0` compose?

Reply:
adding also a RH ranch example makes sense to me 👍